docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.8"
+
+services:
+  modelardbd:
+    container_name: modelardbd
```

Comment:
I am not extremely comfortable with Docker yet but I don't think it is necessary to manually do anything with the `Dockerfile` since we link the service to the `Dockerfile` with `build: .` on the next line.

Reply:
Ok